src/components/Unauthorized/Unauthorized.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from 'react';
+
+import { useNavigate } from 'react-router-dom';
+
+import { DELAY } from '@/constants/constantValues';
+
+export default function Unauthorized() {
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      navigate('/login', { replace: true });
+    }, DELAY);
+
+    return () => clearTimeout(timeoutId);
+  }, []);
+
+  return (
+    <div className='error'>
+      <h1>Unauthorized Access</h1>
+      <h6>Redirecting to login page...</h6>
```

Comment:
Investigate whether using h6 tag is valid from the accessibility point of view. I think that it should be h2. You can always change the font size / any CSS rule.